deployment/modules/aws/tesseract/conformance/outputs.tf

```diff
@@ -0,0 +1,41 @@
+output "s3_bucket_name" {
+  value = module.storage.s3_bucket_name
+}
+
+output "rds_aurora_cluster_endpoint" {
+  value     = module.storage.rds_aurora_cluster_endpoint
+  sensitive = true
+}
+
```

Comment:
These variables are not used, but I think you can use them in Tessera's arguments.

Reply:
Good catch. I've removed all of them except `ecs_cluster` and `vpc_subnets` which need to be used in the next step (Run CT Hammer).